### ISSUE TITLE
Asciidoctor Warnings

### ISSUE THREAD

**Issue by kinueng** [OpenLiberty/draft-guide-microprofile-jwt#9]
These asciidoctor warnings are being thrown when building openliberty.io.  If you need further debugging, let me know.

```
asciidoctor: WARNING: README.adoc: line 135: tag 'doc' not found in include file: /home/travis/build/kinueng/openliberty.io/src/main/content/guides/draft-guide-microprofile-jwt/finish/backendServices/src/main/java/io/openliberty/guides/inventory/InventoryResource.java
asciidoctor: WARNING: README.adoc: line 154: tag 'doc' not found in include file: /home/travis/build/kinueng/openliberty.io/src/main/content/guides/draft-guide-microprofile-jwt/finish/backendServices/src/main/java/io/openliberty/guides/inventory/client/SecuredSystemClient.java
asciidoctor: WARNING: README.adoc: line 172: tag 'doc' not found in include file: /home/travis/build/kinueng/openliberty.io/src/main/content/guides/draft-guide-microprofile-jwt/finish/backendServices/src/main/java/io/openliberty/guides/system/SystemResource.java
asciidoctor: WARNING: README.adoc: line 224: tag 'doc' not found in include file: /home/travis/build/kinueng/openliberty.io/src/main/content/guides/draft-guide-microprofile-jwt/finish/frontendUI/src/main/java/io/openliberty/guides/ui/SystemBean.java
asciidoctor: WARNING: README.adoc: line 243: tags 'copyright,doc' not found in include file: /home/travis/build/kinueng/openliberty.io/src/main/content/guides/draft-guide-microprofile-jwt/finish/frontendUI/src/main/liberty/config/server.xml
```

**Comment by JunyiSun** [comment]
Fixed by #10

**Comment by jamiecoleman92** [comment]
Please close this issue if it has been fixed!
